src/Bundle/CoverallsBundle/Config/Configurator.php

```diff
@@ -112,7 +112,13 @@ protected function createConfiguration(array $options, $rootDir, InputInterface
             }
         }
 
+        $entrypoint = $options['entrypoint'];
+        if (isset($_SERVER['COVERALLS_ENTRYPOINT'])) {
```

Comment:
i believe configuration file is enough. i would like to not rely on globals/env vars

Reply:
Ok, I remove environment variable and leave it only in config